File: gym_be/gym_be/Models/Entities/WorkoutCourse.cs
using gym_be.Models.Entities;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace gym_be.Models.Entities;


[Table("workoutcourse")]
public class WorkoutCourse
{
    [Key]
    [Column("courseid")]
    public Guid CourseId { get; set; }


    [Column("coursename")]
    public string CourseName { get; set; }

    [Column("imageurl")]
    public string ImageUrl { get; set; }

    [Column("personaltrainer")]
    public Guid PersonalTrainerId { get; set; }

    [Column("durationweek")]
    public int DurationWeek { get; set; }

    [Column("description")]
    public string Description { get; set; } 

    
    public Customer PersonalTrainer { get; set; } 
}

File: gym_fe/src/types/workOutCourse.ts
export interface WorkoutCourse {
  courseid: string;
  coursename: string;
  imageurl: string;
  personaltrainer: string;
  durationweek: number;
  description: string;
  trainername?: string;
}


export interface PersonalTrainer {
  id: string,
  name: string,
  specialization: string,
  experience: number
}